src/screens/OnboardingScreen.tsx

```diff
@@ -88,6 +98,26 @@ export const OnboardingScreen = ({
     setActiveIndex(viewableItems[0].index);
   });
   const isLastOnboardingPage = activeIndex === CAROUSEL_ITEMS.length - 1;
+  let backgroundPermission: Location.PermissionResponse;
+
+  const requestPermissions = () => {
+    Alert.alert(
+      "Allow Location Access",
+      "In order to use this app to its full potential, please allow location access. Set location access to 'Always' for the best experience.",
```

Comment:
I think it would be a good idea to workshop the wording for the onboarding page and alert with the team.

Reply:
Yep, we can do this on Tuesday